api/src/main/java/com/discordsrv/api/discord/entity/message/impl/SendableDiscordMessageImpl.java

```diff
@@ -476,6 +477,11 @@ private Function<Matcher, Object> wrapFunction(Function<Matcher, Object> functio
                         ))
                 );
 
+                // Added by Formatter#addContext(Color) under AbstractGameMessageModule#setPlaceholders
+                embedBuilder.setColor(
+                        new Color(placeholders.apply("%color_hex|000001%"))
+                );
```

Comment:
We should add "formattable"-`String` options for color (and timestamp) in `DiscordMessageEmbed` and it's builder, so placeholders can be set directly in to the embed format, rather than a different option